seacrowd/sea_datasets/iatf/iatf.py

```diff
@@ -0,0 +1,182 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pyreadr
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import (TASK_TO_SCHEMA, Licenses, Tasks)
+
+_DATASETNAME = "iatf"
+
+_CITATION = """\
+@misc{
+   iatf,
+   title={Inter-Agency Task Force for the Management of Emerging Infectious Diseases (IATF) COVID-19 Resolutions},
+   url={https://como-ph.github.io/post/creating-text-data-from-iatf-resolutions/},
+   author={Chris Mercado, John Robert Medina, Ernest Guevarra}
+}
+"""
+
+_DESCRIPTION = """\
+To assess possible impact of various COVID-19 prediction models on Philippine government response, text from various resolutions issued by
+the Inter-agency Task Force for the Management of Emerging Infectious Diseases (IATF) has been collected using data mining approaches implemented in R.
+"""
+
+_HOMEPAGE = "https://github.com/como-ph/covidphtext/tree/master/data"
+
+_LICENSE = Licenses.GPL_3_0.value
+
+_SUPPORTED_TASKS = [Tasks.SELF_SUPERVISED_PRETRAINING]
+_SEACROWD_SCHEMA_NAME = TASK_TO_SCHEMA[_SUPPORTED_TASKS[0]].lower()
+_LANGUAGES = ["fil"]
```

Comment:
```suggestion
_LANGUAGES = ["fil"]
_LOCAL = False
```

Adding `_LOCAL`, following metadata specified in `template.py`.

Reply:
updated